### ISSUE TITLE
NER Models not using BIO tagging scheme

### ISSUE THREAD

**Issue by guillaume-be** [huggingface/transformers#4262]
Hello,

I have noticed that most (all?) pre-trained models uploaded seem not to use the BIO tagging scheme, instead returning `I-xxx` and `O` for all categories (`B-xxx` is not used). I believe this is just caused by the way these models have been trained. This makes using these models in actual pipelines difficult as entity parsing becomes impossible (two consecutive entities cannot be differentiated from a multi-tokens entity). 

Are there any plans for updating these models with a BIO labelling scheme?
@stefan-it including you in the discussion as you have uploaded standard models used in the default pipelines

**Comment by stefan-it** [comment]
Hi @guillaume-be , only the CoNLL-2003 models are using this IOB1 tagging scheme, because it is used in the original dataset. You could convert it (+ needs proper testing) but this requires a modification of the original dataset.

I could train models e.g. on the WikiANN corpus (see [here](https://www.aclweb.org/anthology/P17-1178/) and WikiANN with balanced train/dev/test splits [here](https://www.aclweb.org/anthology/P19-1015/)) for English, but this corpus is just a "silver standard" :)

**Comment by guillaume-be** [comment]
Hi @stefan-it ,

Thank you for your response. Assuming the CoNLL-03 model is in IOB1 tagging scheme, it would still use `B-*` labels in case of multi-token entities. For example:
```
Alex I-PER
is O
going O
to O
Los B-LOC
Angeles I-LOC
```
However, the model currently outputs the following for this example:
```
Alex I-PER
is O
going O
to O
Los I-LOC
Angeles I-LOC
```

This means that it is impossible to differentiate multi-token entities (such as Los Angeles) from multiple consecutive entities, e.g. `Alex Bob and Amy are going to Los Angeles`. So far I have never seen the model uploaded generate a `B-*` token.

**Comment by stefan-it** [comment]
`B-*` is only used when an entity follows right after another entity, e.g. different political parties, like this example from the training dataset:

```bash
MAY NNP I-NP O
1996 CD I-NP O
CDU NNP I-NP I-ORG
/ SYM O I-ORG
CSU NNP I-NP I-ORG
SPD NNP I-NP B-ORG
FDP NNP I-NP B-ORG
Greens NNP I-NP B-ORG
PDS NNP I-NP B-ORG
```

Just have a look at the last column. `CDU/CSU` can be considered as one entity, but right after it another political party `SPD` is followed, so it is tagged with a beginning `B-` :)

**Comment by guillaume-be** [comment]
@stefan-it thank you for the clarification!

**Comment by petulla** [comment]
@stefan-it are there plans to merge this per? it could help with this issue. i have it myself as i need to pool entity embeddings. https://github.com/huggingface/transformers/issues/3548

**Comment by mfuntowicz** [comment]
@petulla Should be merged soon 👍
